### ISSUE TITLE
[a11y] API Key Signup page <noscript> tag

### ISSUE THREAD

**Issue by brylie** [NREL/api-umbrella#68]
## Story

As an API Consumer
I need content on the API Key Signup page to be accessible
So that I can understand how to sign up for an API Key
## Observed behavior

The API Key Signup page contains a script to render the signup form. This script is wrapped in <noscript> tags. WebAIM issues a warning with regards to this technique.
## WebAIM details

Content within <noscript> is presented if JavaScript is disabled. Because nearly all users (including users of screen readers and other assistive technologies) have JavaScript enabled, <noscript> cannot be used to provide an accessible version of inaccessible scripted content.

How to Fix It

Ensure that scripted content is accessible. The <noscript> content will be presented to very few users, but must be accessible if used.
## Related Issue

See also #67
